src/JavaScriptSDK.Interfaces/ITelemetryItem.ts

```diff
@@ -6,24 +6,29 @@
  * Telemety item supported in Core
  */
 export interface ITelemetryItem {
+    /**
+     * CommonSchema Version of this SDK
+     */
+    ver?: string;
+
     /**
      * Unique name of the telemetry item
-     */ 
+     */
     name: string;
 
     /**
      * Timestamp when item was sent
-     */ 
-    timestamp?: Date;
+     */
+    time?: Date;
```

Comment:
change this to string. @hectorhdzg , @ramthi  - this is a string per spec. Please see https://1dsdocs.azurewebsites.net/articles/schema/schema-structure.html

Reply:
Correct, I believe we get this value from AppInsights Core, we will try to convert everything to supported types in the channel so Date will become a string before reaching Collector, having a string type since the beginning is obviously better